libs/eo/shared/atomic-design/ui-atoms/src/lib/eo-resolution-picker/eo-resolution-picker.component.ts

```diff
@@ -0,0 +1,133 @@
+import { CommonModule } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  NgModule,
+  Output,
+  ViewEncapsulation,
+} from '@angular/core';
+import { MatButtonToggleModule } from '@angular/material/button-toggle';
+import {
+  AppSettingsStore,
+  Resolution,
+} from '@energinet-datahub/eo/shared/services';
+import {
+  differenceInDays,
+  differenceInMonths,
+  differenceInYears,
+} from 'date-fns';
+import { map } from 'rxjs';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
+  selector: 'eo-resolution-picker',
+  styles: [
+    `
+      eo-resolution-picker {
+        display: block;
+      }
+
+      .title {
+        text-transform: uppercase;
+        display: block;
+      }
+
+      mat-button-toggle-group {
+        border-color: var(--watt-color-primary) !important;
+      }
+
+      mat-button-toggle {
+        border-color: var(--watt-color-primary) !important;
+      }
+
+      mat-button-toggle.mat-button-toggle-appearance-standard
+        .mat-button-toggle-label-content {
+        line-height: unset;
+        padding: 10px 14px;
+      }
+
+      mat-button-toggle:not(.mat-button-toggle-disabled, .mat-button-toggle-checked) {
+        color: var(--watt-color-primary);
+      }
+
+      mat-button-toggle.mat-button-toggle-appearance-standard.mat-button-toggle-checked {
+        background-color: var(--watt-color-primary);
+        color: var(--watt-color-neutral-white);
+      }
+    `,
+  ],
+  template: `<label class="watt-space-stack-s title">Resolution</label>
+    <mat-button-toggle-group
+      [value]="appSettingsResolution$ | async"
+      name="resolution"
+    >
+      <mat-button-toggle
+        [disabled]="
+          !(((appSettingsCalenderDateRangeInDays$ | async) ?? 0) <= 7)
```

Comment:
Did UX want us to make this appear as an option if we were under 3 days, or am i remember it wrong? :)